pnpm-lock.yaml

```diff
@@ -31057,6 +31086,16 @@ packages:
       - supports-color
     dev: true
 
+  /http-proxy-agent/7.0.2:
+    resolution: {integrity: sha512-T1gkAiYYDWYx3V5Bmyu7HcfcvL7mUrTWiM6yOfa3PIphViJ/gFPbvidQ+veqSOHci/PxBcDabeUNCzpOODJZig==}
+    engines: {node: '>= 14'}
+    dependencies:
+      agent-base: 7.1.0
+      debug: 4.3.4
+    transitivePeerDependencies:
+      - supports-color
+    dev: true
+
```

Comment:
Why is this not deduped with http-proxy-agent 7.0.0?

Reply:
pm2 (in `azure/packages`) relies on proxy-agent 6.3 and puppeteer relies on proxy-agent 6.4 which seems to be the main reason for the duplication in versions.

This is the result of bumping the dependency everywhere and running `pnpm i` from the root.